addons/website/static/src/scss/bootstrap_overridden.scss

```diff
@@ -187,3 +187,8 @@ $o-navbar-nav-pills-link-border-radius: if(o-website-value('header-links-style')
 // Accordion
 $accordion-color: inherit !default;
 $accordion-bg: inherit !default;
+
+.row:has(>[class^="col"]) > .border-start,
+.row:has(>[class^="col"]) > .border-end {
+    width:auto
+}
```

Comment:
- We do not use `:has` in CSS because it is not supported everywhere (For example, [the user must activate it himself in the Firefox settings](https://developer.mozilla.org/en-US/docs/Web/CSS/:has#browser_compatibility))

- Also I wouldn't put this in `bootstrap override.css`. In my opinion, it should be somewhere in the CSS of `website_blog` and it should only target the element to be fixed and not the elements with the class `border-end` / `border-start` which are direct children of a row with columns. With this, you would force `width: auto;` on any column with this `border-...` class  :grimacing:

Reply:
@bvr-odoo, Indeed, my idea was to correct all the cases where a `border-start` \ `border-end` are direct children of a row with columns (as this case could occur in other locations than `website_blog`). But if I understand correctly, you would not do that in stable as customer could rely on the behavior without this fix for some customizations?

Reply:
> @bvr-odoo, Indeed, my idea was to correct all the cases where a `border-start` \ `border-end` are direct children of a row with columns (as this case could occur in other locations than `website_blog`). But if I understand correctly, you would not do that in stable as customer could rely on the behavior without this fix for some customizations?

yes this would break any element with this class (e.g. a column :fearful: ) in a row